packages/autocomplete-core/src/types/getters.ts

```diff
@@ -18,9 +18,9 @@ export interface AutocompleteAccessibilityGetters<
 
 export type GetEnvironmentProps = (props: {
   [key: string]: unknown;
-  searchBoxElement: HTMLElement;
-  panelElement: HTMLElement;
+  formElement: HTMLElement;
```

Comment:
is this the form or the form container?

Reply:
The form.

Reply:
should it not be HTMLFormElement then?

Reply:
We could restrict it indeed. It's only used to determine if we should close the panel on scroll so I'm not sure how restrictive we should be: https://github.com/algolia/autocomplete.js/blob/ed0004bc107517c4526491d4b959a483689e5acf/packages/autocomplete-core/src/getPropGetters.ts#L52

Reply:
Do we not add submit and reset event listeners on the form to prevent default? Or is that somewhere else? In any case, it's not really blocking the PR

Reply:
Nope, this is coming from `getFormProps`.